src/docs/api/public.mdx

```diff
@@ -1,76 +1,20 @@
 ---
-title: 'Making an API Public'
+title: 'Publishing an API'
 sidebar_order: 4
 ---
 
 ## Introduction
 
-In this document, we explain both how you make an API public and the considerations to take into
-account before doing so.
-
-## Definitions
-
-### Managed Client
-
-A managed client is one that Sentry controls, such as our frontend. In this type of client, we
-control both which endpoints are used and the attributes used by those endpoints.
-
-### Partially Managed Client
-
-A partially-managed client is one that we only partially control, since we do not control the
-version of the client that is running for our users; for example, our Sentry CLI. While we control
-the endpoints the CLI uses, once the CLI is installed and running in a customer's implementation, we
-don't control the version of the client a user runs.
-
-### External Client
-
-An external client is a client that Sentry does not control; for example, a script written by a
-customer run in their repository, or an integration written using the Integration Platform. For this
-type of client, we have no control over the endpoints they choose to use, nor can we require that
-they immediately update their endpoint.
-
-### Private API
-
-A private API is an undocumented API. Sentry can make breaking changes to this type of API, and no
-one will blink an eye.
-
-### Public API
-
-Our public API is documented at [https://docs.sentry.io/api/](https://docs.sentry.io/api/).
-
-### Private Attributes in a Public Endpoint
-
-You can have private attributes within a public endpoint.
-
-As an example:
-[https://docs.sentry.io/api/teams/retrieve-a-team/](https://docs.sentry.io/api/teams/retrieve-a-team/). 
-The response has multiple attributes:
-
-```json
-{
-  "id": "2",
-  "slug": "the-interstellar-jurisdiction"
-  ...
-}
-```
-
-Let's say we also return a `nickname` for a team. If what is returned is not documented as a
-response in our public documentation, that attribute is a private attribute within a public
-endpoint.
-
-### DRF (Django REST Framework)
```

Comment:
Added this to a different page here: https://github.com/getsentry/develop/pull/1032
I don't think it's related to publishing